experiments/models/appointment_models.py

```diff
@@ -105,12 +107,26 @@ def __str__(self):
             self.experiment.name
         )
 
+    @property
     def start(self):
         return self.timeslot.start
 
+    @start.setter
+    def start(self, starttime):
+        self.timeslot.start = starttime 
+        self.timeslot.save()
```

Comment:
maybe the call to timeslot.save() should only be made when Appointment is saved?